sqlcollaborative/dbatools
# New commands agent output files

Fixes # 

Changes proposed in this pull request:
Two New Commands

Get-SQLAgentJobOutputfile and
Set-SQLAgentJobOutputfile

How to test this code: 

Run against a server with Agent Jobs with or without output files
Run against a server with Agent Jobs with multiple job steps 
Check values or changes are valid

Has been tested on minimum requirements:
- [ x]  Powershell 3
- [ X]  Windows 7
- [x ]  SQL Server 2000

Has been tested on maximum requirements:
- [ ]  SQL Server vNext
- [x ]  Windows 10
- [ N/A]  Azure Database

Tests for tester:
Not required
Does Not support multiple servers - Not required IMO happy to discuss
